awscli/examples/cloudtrail/put-event-selectors.rst

```diff
@@ -0,0 +1,25 @@
+**To configure event selectors for a trail**
+
+The following ``put-event-selectors`` command configures an event selector for ``Trail1``::
+
+  aws cloudtrail put-event-selectors --trail-name Trail1 --event-selectors '[{ "ReadWriteType": "All", "IncludeManagementEvents":true, "DataResources": [{ "Type": "AWS::S3::Object", "Values": ["arn:aws:s3:::mybucket/prefix"] }] }]'electors --trail-name Trail1

```

Comment:
At the end of ``--event-selectors`` there is this ``electors`` value that does not look to be a part of the JSON document provided to the parameter. Is this intentional?